build.sbt

```diff
@@ -20,7 +20,8 @@ lazy val ComponentTest = config("component") extend Test
 
 def oneForkedJvmPerTest(tests: Seq[TestDefinition]): Seq[Group] = {
   tests map {
-    test => Group(test.name, Seq(test), SubProcess(ForkOptions(runJVMOptions = Seq("-Dtest.name=" + test.name))))
+    //test => Group(test.name, Seq(test), SubProcess(ForkOptions(runJVMOptions = Seq("-Dtest.name=" + test.name))))
+    test => Group(test.name, Seq(test), SubProcess(ForkOptions()))
```

Comment:
Fork options has a different signature right now, I change this to default one. I'm not sure if we should use this method. The best option is probably to remove it, but wasn't sure if it's important